OWML.ModHelper.Menus/ModMenu.cs

```diff
@@ -61,6 +68,16 @@ public IModTitleButton GetButton(string title)
             return button;
         }
 
+        public IModPromptButton GetPromptButton(string title)
+        {
+            var button = PromptButtons.FirstOrDefault(x => x.Title == title || x.Button.name == title);
+            if (button == null)
+            {
+                OwmlConsole.WriteLine("Warning: no button found with title or name: " + title);
+            }
+            return button;
```

Comment:
This could now be extracted to a new private method `GetButton(string title, List<IModButton> buttons)` which can be used by GetTitleButton and GetPromptButton

Reply:
K